src/app/features/current-account/current-account-name.tsx

```diff
@@ -26,10 +26,10 @@ const AccountNameSuspense = memo((props: BoxProps) => {
   const currentAccount = useCurrentStacksAccount();
   const name = useCurrentAccountDisplayName();
   if (!currentAccount || typeof currentAccount.index === 'undefined') return null;
```

Comment:
I added this comment so I can flag the code here. It seems a bit complicated - altering the name based on a char limit. 

We could do this using CSS and setting a max width of the account name